src/App.tsx

```diff
@@ -8,7 +8,100 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [user, setUser] = useState<User | 0>(0);
+  const [selectedId, setSelectedId] = useState(0);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | 0>(0);
+  const [todoSortByData, setTodoSortByData] = useState('');
+  const [searchQuery, setSearchQuery] = useState('');
+  const [listLoader, setListLoader] = useState(false);
+  const [modalLoader, setModalLoader] = useState(false);
```

Comment:
```suggestion
  const [islistLoading, setIsListLoading] = useState(false);
  const [isModalLoading, setIsModalLoading] = useState(false);
```
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig